worker/worker-manager/score-worker-manager-impl/src/main/java/io/cloudslang/worker/management/services/WorkerRecoveryManagerImpl.java

```diff
@@ -23,9 +23,10 @@
  */
 public class WorkerRecoveryManagerImpl implements WorkerRecoveryManager {
 
-	protected static final Logger logger = Logger.getLogger(WorkerRecoveryManagerImpl.class);
+    protected static final Logger logger = Logger.getLogger(WorkerRecoveryManagerImpl.class);
+    private static final int EXIT_STATUS = 111;
```

Comment:
@natabeck sorry for this BS but mo OCD can't give up....

We can go with 75 - this is as close to a standart as I could find.
from: [/usr/include/sysexits.h](http://www.opensource.apple.com/source/Libc/Libc-320/include/sysexits.h)
```

#define EX_TEMPFAIL	75	/* temp failure; user is invited to retry */
 *	EX_TEMPFAIL -- temporary failure, indicating something that
 *		is not really an error.  In sendmail, this means
 *		that a mailer (e.g.) could not create a connection,
 *		and the request should be reattempted later.
```